apps/admin/frontend/src/screens/write_ins_adjudication_screen.tsx

```diff
@@ -373,32 +382,60 @@ export function WriteInsAdjudicationScreen(): JSX.Element {
       !!currentWriteInId
     );
 
-  const [doubleVoteAlert, setDoubleVoteAlert] = useState<DoubleVoteAlert>();
-  const [showNewWriteInCandidateForm, setShowNewWriteInCandidateForm] =
-    useState(false);
-  const newWriteInCandidateInput = useRef<HTMLInputElement>(null);
-  const nextButton = useRef<Button>(null);
-  const firstAdjudicationButton = useRef<Button>(null);
+  // sets the user's position in the adjudication clue to the first pending write-in
+  useEffect(() => {
+    if (
+      firstPendingWriteInIdQuery.isSuccess &&
+      writeInQueueQuery.isSuccess &&
+      !initialOffsetSet
+    ) {
+      const firstPendingWriteInId = firstPendingWriteInIdQuery.data;
+      const writeInQueue = writeInQueueQuery.data;
+      if (firstPendingWriteInId) {
+        setOffset(writeInQueue.indexOf(firstPendingWriteInId));
+      }
+      setInitialOffsetSet(true);
+    }
+  }, [firstPendingWriteInIdQuery, writeInQueueQuery, initialOffsetSet]);
 
-  // prefetch the next write-in image
+  // prefetch the next and previous write-in image
   useEffect(() => {
```

Comment:
This part - these two dependent effects and the dependent queries - is probably the most fragile part of the whole adjudication page. I'm not quite sure how to make less side-effect-y, but it seems reliable in manual and automated testing.